src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/utils/MapUtils.java

```diff
@@ -0,0 +1,23 @@
+package altinn.platform.pdf.utils;
+
+import java.util.Map;
+
+public class MapUtils {
+  private MapUtils() {
+  }
+
+  public static String getLabelFromValue(Map<String, Map<String,String>> dict, String outerKey, String innerValue){
```

Comment:
A simple test for this method would also be neat! 🗡️

Reply:
Done :)  but doesn't seem to be run by the pipeline or picked up by SonarCloud?